lib/sa.ts

```diff
@@ -0,0 +1,93 @@
+import { expect, Page } from "@playwright/test";
+import { config } from "./config";
+
+export const navigateToSAList = async function (page: Page) {
+    await page.getByRole('link', { name: 'Application and Data Services' }).click();
+    // }
+    await page.getByRole('link', { name: 'Service Accounts' }).click();
+    await expect(page.getByRole('heading', { name: 'Service Accounts' })).toHaveCount(1);
+};
+
+export const createServiceAccount = async function (page: Page, name: string) {
+    await page.locator('button', { hasText: 'Create service account' }).click();
+
+    await expect(page.getByText('Create a service account', {exact: true})).toHaveCount(1);
```

Comment:
can you use something like `await page.locator('button', { hasText: 'Create service account' }).click();` ? From my POV it should work